docs/community/documentation-contribution-guide.rst

```diff
@@ -0,0 +1,339 @@
+To do X, click Y                         Click Y, to do X
+=======================================  =====================================
+
+- avoid using "let's..." constructions
+- avoid using "simply", "it's very easy" or similar phrases
+- avoid using internet slang, for example "TL;DR"
+- avoid using "please", as in "please click the button"
+- avoid abbreviations, as the readers may not be familiar with them
+- do repeat nouns instead of using pronouns, to avoid confusion:
+
+======================================================================   ===============================================================
+Do                                                                       Don't
+======================================================================   ===============================================================
+if you create a file in a specific bucket, make sure the file is named   if you create a file in a specific bucket, make sure it's named
+======================================================================   ===============================================================
+
+Technical writing style guidelines
+----------------------------------
+Remind yourself that the person reading your documentation may have no
+knowledge of what the module does.
+Always give a general, brief overview of any new term and explain its
+function and purpose.
+Remember to be consistent, do not use different words for the same
+concept throughout the text.
+
+Highlighting and formatting
+---------------------------
+Formatting guidelines:
+
+- use camel case or snake case where appropriate and do not split
+  object/function names into separate words.
+- use angle brackets for placeholders
+- do not include the command line prompt ($) when writing commands
+- separate commands from their output
+- use the :code: tag for commands, objects, field names, filenames,
+  directories, paths, etc.
+- use the :py:function: tag along with the fully qualified name of the function
+  when referencing to also make link to the API docs
+
+============================================  ===============================
+Do                                            Don't
+============================================  ===============================
+:code:`:py:func:`invenio.app.function_name``  :code:`:code:`function_name``
+:code:`:py:class:`invenio.__init__``          :code:`invenio.__init`
+:code:`:py:data:`invenio.config.DATA``        :code:```invenio.config.DATA```
+============================================  ===============================
+
+- use normal style for string and integer field values
+
+Code samples
+------------
+- keep a maximum column width of 80 for every code block in order not to
+  have any horizontal scrolling
+- Use ``doctest`` whenever possible to make sure that the documented code
+  works. If not possible use simple code blocks.
+
+Diagrams
+--------
+For drawing diagrams, you can use draw.io. It provides you with the
```

Comment:
The part where you make reference to where to put the diagrams source/PNG, would be more clear with a tree structure? :)